website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
+            [
+                '/project/<pid>/registration/approve/<token>/',
+                '/project/<pid>/node/<nid>/registration/approve/<token>/',
+            ],
+            'get',
+            project_views.register.node_registration_approve,
+            OsfWebRenderer('error.mako', trust=False)
```

Comment:
All `HTTPErrors` get rendered using `error.mako` anyway: https://github.com/samchrisinger/osf.io/blob/09f7f402630df2151ee36b402b40546bf17214eb/framework/routing/__init__.py#L397

I think `notemplate` is what you want here: https://github.com/samchrisinger/osf.io/blob/09f7f402630df2151ee36b402b40546bf17214eb/website/routes.py#L90-L91